Sources/TSCUtility/Platform.swift

```diff
@@ -25,6 +26,9 @@ public enum Platform {
     public static var currentPlatform = Platform.findCurrentPlatform()
     /// Attempt to match `uname` with recognized platforms.
     private static func findCurrentPlatform() -> Platform? {
+      #if os(Windows)
+        return .windows
+      #else
         guard let uname = try? Process.checkNonZeroExit(args: "uname").spm_chomp().lowercased() else { return nil }
```

Comment:
Is there a reason to dynamically do this check? @aciidb0mb3r

Reply:
Not that I know of. I don't remember why we have the API and maybe can get rid of it?

Reply:
I don't see a reason for this either; it seems to me that `uname` should always return "Darwin" if and only if `os(Darwin)` is true.

Reply:
What I didn't realize when I wrote my previous comment was that there isn't actually a `os(Darwin)`, and I don't think that spelling out the current set of Darwin-based OSs is the right approach.  `canImport(Darwin)` would work, but I'd be a little nervous about assuming that only Darwin platforms have the Darwin module.  So maybe best to just ignore my comment and keep calling `uname` for Darwin.